### PR TITLE
{chem}[intel/2015a] libbeef 0.1.1 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/l/libbeef/libbeef-0.1.1-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['https://confluence.slac.stanford.edu/download/attachments/146704476']
+sources = [SOURCE_TAR_GZ]
+
+configopts = 'CC="$CC"'
+
+sanity_check_paths = {
+    'files': ['bin/bee', 'lib/libbeef.a'],
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/libbeef/libbeef-0.1.1-intel-2015a.eb]: `bee` or `beef`?

Comment (boegel) [inline easybuild/easyconfigs/l/libbeef/libbeef-0.1.1-intel-2015a.eb]: The binary is named `bee`, the library is named `beef`. I don't question, I just observe. ;-)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b48874649f4fc7527c86 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**